### PR TITLE
fix: login via metamask with default network

### PR DESCRIPTION
**Pull request by IgorShadurin**
Close https://github.com/fairDataSociety/fairdrive-theapp/issues/425

### CODE REVIEW

File: src/utils/localStorage.ts [hunk 1]
```diff
@@ -2,3 +2,18 @@ export enum LocalStorageKeys {
...
+ * Sets default network to local storage
+ * @param network Network id
+ */
+export function setDefaultNetwork(network: string): void {
+  localStorage.setItem(LocalStorageKeys.NETWORK, network);
+}
+
+/**
+ * Gets default network from local storage
+ */
```
Comment (tomicvladan) [inline src/utils/localStorage.ts:16]: There is already `getDefaultNetwork` function in `src/context/FdpStorageContext.tsx`. It prevents errors from server side rendering and also provides a default network when localStorage is empty.

Comment (IgorShadurin) [reply src/utils/localStorage.ts:16]: The purpose of this function is to have all work with localStorage come from localStorage utils. In this case, there are 2 getDefaultNetwork functions, one top level that checks the server side and one that gets the value from localStorage.

The rest of the accesses should also be moved to localStorage utils, but I only move what is relevant to this task.

Comment (tfius) [reply src/utils/localStorage.ts:16]: why ? doesn't this bring unnecesary complexity ?

Comment (tomicvladan) [reply src/utils/localStorage.ts:16]: When a user visits the app for the first time, there will be no entries for default network. That means all components will receive null as default network? The function should provide a default value in that case.

Comment (IgorShadurin) [reply src/utils/localStorage.ts:16]: getDefaultNetwork in FdpStorageContext.tsx works the same as it did before. I just moved the code that changes localStorage to helper to keep all changes to storage under control.

Now different components change different values in localStorage and control over these functions is gradually lost. In order to answer the question during development, someone changes or receives some value from storage, you will need to check all the components in the entire project.

By moving these methods to helper, it is easier to control the situation.

**Review by tomicvladan — APPROVED** [review]

**Pull request merged**